harambe_scrapers/extractor/wayne_commission/listing.py

```
@@ -0,0 +1,358 @@
+                    "(KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36 Edg/138.0.0.0"
+                ),
+                "x-b3-sampled": "1",
+                "x-b3-spanid": "8637d509111f36c5",
+                "x-b3-traceid": "b1fb5355028f0b97b4d23cd37b51d7ee",
+                "x-oracle-apm-ba-version": "1.1.131",
+                "x-requested-with": "XMLHttpRequest",
+                "Cookie": "ASP.NET_SessionId=h4ooyutzz33houczonxywo0h",
+            }
+
+            print(f"[LISTING] Making API POST to {url}")
+            response = requests.request("POST", url, headers=headers, data=payload)
```

Ruff (0.14.6)

Probable use of `requests` call without timeout

(S113)